README.md

```diff
@@ -88,6 +88,7 @@ Please check the [contribution guidelines](CONTRIBUTING.md) for info on formatti
 
 ## Tools
 #### General
+- [Superblocks Lab](https://lab.superblocks.com/) - Web IDE. Built in browser blockhain VM, Metamask integration (one click deployments to Testnet/Mainnet), transaction logger and live code your WebApp among many other features.
```

Comment:
Two minor points:
* To keep an alphabetical ordering, please move your entry under `solidity-docgen`.
* The description is hard to scan quickly, I think something along the lines of "Web IDE with built-in blockchain VM, Metamask integration, transaction logger, and live coding." would be more accessible and shorter.